packages/chakra-ui-core/src/CLink/accessibility.md

```diff
@@ -0,0 +1,24 @@
+# Link | Accessibility ♿️
+
+This report is adapted to list the [WAI-ARIA Authoring practices for alert dialogs](https://www.w3.org/TR/wai-aria-practices-1.2/#link) supported by Chakra UI for the `CLink` component.
+
+### Description
+A link widget provides an interactive reference to a resource. The target resource can be either external or local, i.e., either outside or within the current page or application.
+
+#### Components
+`@chakra-ui/vue` exports 1 checkbox related components:
+- `CLink`
+
+### `CLink` Keyboard Interaction
+- [x] **`Enter`**: Executes the link and moves focus to the link target.
+- [x] **`Shift + F10 (Optional)`**: Opens a context menu for the link.
```

Comment:
Since this is optional and not implemented on the CLink component I think we should remove it so readers aren't mislead. 👍

Reply:
Sure @codebender828. I will remove it. 

Thanks a lot for you valuable feedback. i will take care all of this in future.